sale_order_line_case_number/tests/test_sale_order_line_case_number.py

```diff
@@ -0,0 +1,54 @@
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrderLineCaseNumber(TransactionCase):
+    def setUp(self):
+        super(TestSaleOrderLineCaseNumber, self).setUp()
+        self.product1 = self.env["product.template"].create(
+            {
+                "name": "Product1",
+                "list_price": 100.00,
+                "default_code": "Test",
+                "case_number": "123",
+            }
+        )
+        self.product2 = self.env["product.template"].create(
+            {
+                "name": "Product2",
+                "list_price": 100.00,
+                "default_code": "Test2",
+                "case_number": "123",
```

Comment:
```suggestion
                "case_number": "456",
```
To make the intention clearer.